### DISCUSSION TITLE
Any way to exclude notes using Option Query Argument?

### DISCUSSION THREAD

**Discussion by bykiandra** [mProjectsCode/obsidian-meta-bind-plugin#154] [Q&A] [answered]
So I'm using inline list suggester with the option query argument set to a particular tag. The problem is that the notes I'm searching through are created from a template, which also has the same tag.

Is there some way I can exclude the template file from the suggestions?

**Comment by mProjectsCode** [comment]
you can [combine sources](https://blacksmithgu.github.io/obsidian-dataview/reference/sources/#combining-sources) and exclude your template folder.

**Reply by bykiandra** [reply]
Would you mind giving an example please? I've tried every sort of combination I can think of from the link you gave, and I keep getting errors. Maybe I'm not putting in the correct path for the folder?

**Reply by mProjectsCode** [reply, marked as answer]
`optionQuery(#yourTag and !"path/to/template/folder")`
